indico/modules/events/editing/controllers/backend/management.py

```diff
@@ -48,6 +53,18 @@ def _process_DELETE(self):
         return '', 204
 
 
+class RHCreateFileType(RHEditingManagementBase):
+    """Create a new file type."""
+
+    @use_rh_args(EditableFileTypeArgs)
+    def _process(self, data):
+        for extension in data.get('extensions', []):
+            if re.match(r'^\.', extension):
+                raise ExpectedError(_('Extensions cannot have leading dots'))
```

Comment:
This should be probably replaced with:

```python
from webargs.flaskparser import abort
...
abort(422, messages={'extensions': [_('Extensions cannot have leading dots')]})
```

this way, users will see a nice error message attached to the field. At the moment nothing indicates what caused the error.

Reply:
or better, add a validation method for the field in EditableFileTypeArgs